nimble/host/services/dis/syscfg.yml

```diff
@@ -0,0 +1,116 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Package: nimble/host/services/dis
+
+syscfg.defs:
+    BLE_SVC_DIS_MODEL_NUMBER_READ_PERM:
+        description: >
+            Defines permissions for reading "Model Number" characteristics. Can
+            be zero to allow read without extra permissions or combination of:
+                BLE_GATT_CHR_F_READ_ENC
+                BLE_GATT_CHR_F_READ_AUTHEN
+                BLE_GATT_CHR_F_READ_AUTHOR
+            Set to '-1' to remove this characteristic.
+        value: -1
```

Comment:
this is c&p multiple times - I'd make it smarter by adding syscfg val e.g. BLE_SVC_DIS_DEFAULT_READ_PERM with all the description there and then just reference it in descriptions of all other characteristics. additionally, it may be set to 0 to allow read and value of this particular characteristic could be set to MYNEWT_VAL_BLE_SVC_DIS_DEFAULT_READ_PERM as an example so it inherits that common value. other characteristics can be -1 to be disabled by default.